tests/sundials/n_vector.cc

```diff
@@ -355,8 +355,10 @@ test_get_communicator()
                                     global_nvector_context
 #endif
   );
+#if !DEAL_II_SUNDIALS_VERSION_GTE(7, 0, 0)
```

Comment:
```suggestion
#if DEAL_II_SUNDIALS_VERSION_LT(7, 0, 0)
```
I don't have a preference on this one. You decide :-)